main.go

```diff
@@ -168,6 +168,13 @@ func main() {
 	// Subscribe to LND invoice updates in the background
 	go svc.InvoiceUpdateSubscription(context.Background())
 
+	//Start webhook subscription
+	if svc.Config.WebhookUrl != "" {
+		webhookCtx, cancelWebhook := context.WithCancel(context.Background())
+		go svc.StartWebhookSubscribtion(webhookCtx)
```

Comment:
from the implementation of `StartWebhookSubscription` this could be called multiple times? 
if so could the URL also passed in as argument?

Reply:
Right now this is only called a single time, here in `main.go`. But we could call it once per webhook in the future. This could be okay in case there is a limited amount of webhooks (probably the case). The other option is to have a single subscription/goroutine for the webhook system, and call the `postToWebhook` multiple times.

Reply:
👍